browse/models.js

```diff
@@ -328,14 +328,19 @@ class BrowseViewModel {
       if (c) c.desiredVisible = false;
     });
     if (this.getKeywordList().length) {
-      Charity.invisibleCharities.forEach((c) => {
-        if (
-          !this.shouldHide(c.id) &&
-          c.searchMatch(Object.keys(this.keywords)) &&
-          !c.desiredVisible
-        ) {
-          c.place();
-        }
+      const matches = Charity.invisibleCharities.filter((c) =>
```

Comment:
My original code is there to make things like "islam" work. Feature request will probably be to add :limit as an option to the keyword. What do you think?